src/ui/jest.config.js

```diff
@@ -52,6 +52,7 @@ module.exports = {
     [`node_modules/(${esModules}).*\\.jsx?$`]: './jest-esm-transform',
     '^.+\\.toml$': 'jest-raw-loader',
   },
+  coverageProvider: 'v8',
```

Comment:
I'm a bit surprised that bypassing Babel in coverage reporting doesn't materially change results. Shouldn't it?

Reply:
the lack of sourcemaps was the issue. fixed.